README.zh-CN.md

```diff
@@ -18,13 +18,20 @@
  - [X] 可扩展空岛领地范围（需要付出一些资源/金钱需要经济插件支持）
  - [X] 可单独创建一个空岛世界与其他世界分离
  - [X] 自定义岛屿重生点或者tp点
- - [X] 利用GriefPrevention的/trust命令来创建一个空岛队伍
+ - [X] 团队功能：
+   - [X] 邀请朋友到岛上
+   - [X] 踢出一个在岛上的玩家
+   - [X] 离开一个岛
+   - [X] 转赠一个岛屿(移交财产么?2333)
+   <!--did your mean it is the dead money? -->
```

Comment:
I have no idea what you are saying here.

Reply:
did you mean the island like something giveaway?or inheritance?

Reply:
Islands have a single owner who has full control over the said island. Transfering ownership means giving that control to somebody else.

Reply:
and than the new owner kicked you out?ROFL.

Reply:
and than the new owner kicked you out?ROFL.

Reply:
and than the new owner kicked you out?ROFL.

Reply:
and than the new owner kicked you out?ROFL.

Reply:
and than the new owner kicked you out?ROFL.

Reply:
and than the new owner kicked you out?ROFL.

Reply:
and than the new owner kicked you out?ROFL.

Reply:
and than the new owner kicked you out?ROFL.

Reply:
and than the new owner kicked you out?ROFL.

Reply:
and than the new owner kicked you out?ROFL.

Reply:
and than the new owner kicked you out?ROFL.

Reply:
@subtank more than likely if you're transferring ownership, you intend to leave the island.